railties/lib/rails/command/helpers/editor.rb

```diff
@@ -28,6 +28,8 @@ def catch_editing_exceptions
             say "Aborted changing file: nothing saved."
           rescue ActiveSupport::EncryptedFile::MissingKeyError => error
             say error.message
+          rescue ActiveSupport::EncryptedFile::MissingKeyError => error
```

Comment:
This is the same exception than above. Should we just change the above line?

Reply:
You are right, my mistake. Actually, it should be a different exception, but still, it can be grouped with the previous one.